src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -882,6 +883,17 @@ public void onDeath(final DamageSource par1DamageSource)
         super.onDeath(par1DamageSource);
     }
 
+    @Override
+    public EnumActionResult applyPlayerInteraction(
+            final EntityPlayer player,
+            final Vec3d vec,
+            @Nullable final ItemStack itemInHand,
+            final EnumHand hand)
+    {
+        SoundUtils.playInteractionSoundAtCitizenWithChance(CompatibilityUtils.getWorld(this), this.getPosition(), 100, this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)